test/unittest/layers/layers_golden_tests.cpp

```diff
@@ -0,0 +1,206 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file layer_golden_tests.cpp
+ * @date 09 Sept 2021
+ * @brief Common golden test for nntrainer layers (Param Tests)
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+#include <layers_common_tests.h>
+
+#include <fstream>
+#include <type_traits>
+
+#include <base_properties.h>
+#include <layer_context.h>
+#include <layer_devel.h>
+#include <nntrainer_test_util.h>
+#include <util_func.h>
+#include <var_grad.h>
+#include <weight.h>
+
+#pragma GCC diagnostic ignored "-Wunused-local-typedefs"
```

Comment:
I think svace/coverity will not like this or the warnings this will ignore.

Reply:
I checked, this is false positive in gcc, so it's suppressed. I agree that the svace/coverity won't like... I have to tell them to have a heart for once.